tests/integration/index/docs.go

```diff
@@ -216,7 +216,7 @@ func getUserDocs() predefined.DocsList {
 					},
 					{
 						"model": "Playstation 5",
-						"year":  2022,
+						"year":  2021,
```

Comment:
question: Why has this changed?

Reply:
so that this query can fetch it:
```graphql
query {
  Device(filter: {
    year: {_eq: 2021}
  }) {
    model
    owner {
      name
    }
  }
}
```
And also to make it consistent with another "Playstation 5".
It's not used in other tests anyway.